src/Microsoft.AspNet.Mvc.Razor/RazorTextWriter.cs

```diff
@@ -54,7 +59,7 @@ public override Encoding Encoding
         private TextWriter TargetWriter { get; set; }
 
         [RazorInject]
```

Comment:
Oops for got to delete that. This was **never** working. `[RazorInject]` only applies directly on the page itself.